versioned_docs/version-latest/learn/monitoring.md

```diff
@@ -0,0 +1,723 @@
+---
+title: Monitoring
+sidebar_position: 4
+description: Monitoring server with Grafana and Prometheus. Scraping metrics from Subspace Node.
+keywords:
+  - Monitoring
+  - Grafana
+  - Prometheus
+  - Exporter
+  - Metrics
+  - Docker
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import Link from '@docusaurus/Link';
+import styles from '@site/src/pages/index.module.css';
+
+
+-----------
+
+### Setting Up Monitoring Infrastructure
+
+**Infrastructure Setup:**
```

Comment:
```suggestion
**Infrastructure Setup Overview:**
```

Reply:
Of course the correct one is under 'learn' sect.

Reply:
Can we remove the other one please?